prow/jobs.md

```diff
@@ -70,7 +70,90 @@ that matches `trigger` will suffice. This is useful if you want to make one
 command that reruns all jobs. If unspecified, the default configuration makes
 `/test <job-name>` trigger the job.
 
-### Pod Utilities
+## Standard Triggering and Execution Behavior for Jobs
+
+When configuring jobs, it is necessary to keep in mind the set of rules Prow has
+for triggering jobs, the GitHub status contexts that those jobs provide, and the
+rules for protecting those contexts on branches.
+
+### Triggering Jobs
+#### Trigger Types
+
+`prow` will consider three different types of jobs that run on pull requests
+(presubmits):
+
+ 1. jobs that run unconditionally and automatically. All jobs that set
+     `always_run: true` fall into this set.
+ 2. jobs that run conditionally, but automatically. All jobs that set
+    `run_if_changed` to some value fall into this set. 
+ 3. jobs that run conditionally, but not automatically. All jobs that set
+    `run_always: false` and do not set `run_if_changed` to any value fall
```

Comment:
I really hope you mean `always_run` instead of `run_always`, and not that we have both of these values.

Reply:
My brain is literally incapable of getting that straight

Reply:
I think this is because the Go value is backwards?